src/recode.jl

```diff
@@ -36,6 +36,28 @@ recode!(dest::CategoricalArray, src::AbstractArray, pairs::Pair...) =
 recode!(dest::CategoricalArray, src::CategoricalArray, pairs::Pair...) =
     recode!(dest, src, nothing, pairs...)
 
+
+"""
+    recode_in(x, collection)
+
+Helper function to test if x is a member of the collection.
+
+If collection is a regular collection-type without missing elements the 
+`in`-function is used. If collection contains a missing element, every
+element is tested using the isequal function. This ensures that missing
+is treated as a valid value.
+"""
+@inline recode_in(x, ::Missing) = false
```

Comment:
It would make sense to add a method for `collection::Set`, since these use `isequal` to test containment, which is much faster than using `any`.

Reply:
Yes, that would be a nice change. I will add it. I can across the different behavior of sets when struggling with missings, but I didn't think of it.